skbio/stats/ordination/_principal_coordinate_analysis.py

```diff
@@ -44,10 +39,26 @@ def pcoa(distance_matrix):
     species is present in two sites, that means that the sites are
     similar.).
 
+    Note that the returned eigenvectors are not normalized to unit length.
+
     Parameters
     ----------
-    distance_matrix : DistanceMatrix
+    matrix_data : DistanceMatrix
         A distance matrix.
+    method : str
+        Eigendecomposition method to use in performing PCoA.
+        By default, uses SciPy's "eigh", which computes exact
+        eigenvectors and eigenvalues for all dimensions. The alternate
+        method, "fsvd", uses faster heuristic eigendecomposition but loses
+        accuracy. The magnitude of accuracy lost is dependent on dataset.
```

Comment:
the reference for fsvd should be cited in this docstring. I know you mentioned it in `_fsvd()`, but it should be here too so the users can see it in the public API documentation